### PR TITLE
Django19 compatibility

### PR DESCRIPTION
**Pull request by maennel**
An attempt of getting CleanerVersion to work together with Django1.9 (as addressed in #111). This branch allows for Django1.9 only. It is not compatible with earlier versions of Django (<1.9) and has not been tested with later versions (>1.10).
Tests pass so far.
It looks like the migration issue on M2M fields (#56) persists.

### CODE REVIEW

**Comment by maennel** [comment]
One major issue remaining in this PR is the subject of migrating M2M relationships. We've noticed the incompatibility of CleanerVersion and Django already in earlier versions of Django. However, with Django 1.9 errors already appear while "making migrations", which makes it unsafe to release this PR as stable.
This refers to #56 and the newly skipped test under `test_migrations`.

**Comment by maennel** [comment]
@brki with the latest changes, we're now at the same state as with previous CV versions:
- Migration creation passing
- Migrations in general passing with the exception of M2M migrations (addressed in #56)

I suggest to merge the code and to release a new major version of CV supporting Django 1.9 and upwards.

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+six>=1.10.0
```
Comment (maennel) [inline requirements.txt]: Usually, six comes with Django: ``django.utils.six``. Adapt...

**Comment by brki** [comment]
Perhaps a thin custom database backend with some custom methods in it's schema module will be necessary for m2m migrations to work.  wdyt @maennel ?

**Comment by ohenrik** [comment]
What is stopping this request from being merged? Supporting Django 1.9 and 1.10 seems important for this package to be used in future projects. Are there any major issues with this pull request?

**Comment by gabn88** [comment]
Any news on the progress? This seems like the best package so far for version control, because adding extra columns is more elegant than adding another table with json serialized fields... ;)

Would love to use this on Django 1.9! 👍

**Comment by techdragon** [comment]
I'm pretty eager to use this in a project but I'm not fond of supporting my own forked versions of libraries. Is there any way I can help get this merged? Is there work remaining to be done?

**Comment by mendozao** [comment]
What needs to happen to get this PR merged? I've been testing `maennel:django19-compatibility` on Django 1.10 and it seems to be working well

**Comment by maennel** [comment]
Please consider #140 rather than this PR. #140 extends this PR with support for current Django versions (i.e. 1.10 and 1.11).

Closing this PR in the process.

**Comment by maennel** [comment]
Hey everyone, in the meantime, I've released v2.0.0 of CleanerVersion, adding support for Django 1.9, 1.10 and 1.11. I hope this helps you in writing fantastic applications :) 
Appreciate your interest and looking forward to your future feedback. 
Thanks

**Pull request closed without merging**